index.adoc

```diff
@@ -1,178 +1,97 @@
-|
+== Build & Deploy WildFly on the Cloud
 
-|link:29[29.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
-|
+There are different tools to build and deploy WildFly on the Cloud, 
 
-|link:28[28.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
-|
+=== WildFly Container
 
-|link:27[27.0.0.Final]
-|https://jakarta.ee/specifications/platform/10/apidocs/[Jakarta EE 10]
-|
+Our original container image that provides a standalone WildFly server. 
+Use this image to add your deployments and build your application image.
 
-|link:26.1[26.1.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8] (and https://jakarta.ee/specifications/platform/9.1/apidocs/[EE 9.1] Preview)
-|
+Learn more about link:wildfly-container[WildFly Container].
 
-|link:26[26.0.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8] (and https://jakarta.ee/specifications/platform/9.1/apidocs/[EE 9.1] Preview)
-|
+=== Helm Chart
 
-|link:25[25.0.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8] (and https://jakarta.ee/specifications/platform/9.1/apidocs/[EE 9.1] Preview)
-|
+Build & Deploy your Java applications with WildFly on Kubernetes and OpenShift with a Helm Chart.
 
-|link:24[24.0.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8] (and https://jakarta.ee/specifications/platform/9.1/apidocs/[EE 9.1] Preview)
-|
+Learn more about the link:wildfly-charts[Helm Chart for WildFly].
 
-|link:23[23.0.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8] (and https://jakarta.ee/specifications/platform/9/apidocs/[EE 9] Preview)
-|
+=== Operator for Kubernetes & OpenShift
 
-|link:22[22.0.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8] (and https://jakarta.ee/specifications/platform/9/apidocs/[EE 9] Preview)
-|https://javaee.github.io/javaee-spec/javadocs[Java EE 8]
+The WildFly Operator simplifies deployment and maintenance of WildFly containerized applications
+for Kubernetes ans OpenShift.
 
-|link:21[21.0.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8]
-|https://javaee.github.io/javaee-spec/javadocs[Java EE 8]
+Learn more about the link:wildfly-operator[WildFly Operator].
 
-|link:20[20.0.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8]
-|https://javaee.github.io/javaee-spec/javadocs[Java EE 8]
+===  Source-to-Image (S2I) Builder & Runtime Images
 
-|link:19.1[19.1.0.Final]
-|https://jakarta.ee/specifications/platform/8/apidocs/[Jakarta EE 8]
-|https://javaee.github.io/javaee-spec/javadocs[Java EE 8]
+Use Source-to-Image (S2I) Builder image to build a container image on OpenShift directly from your Git repository.
+Use the Runtime image to trim the container so that it contains the minimal dependencies required to run {wildfly-latest-version}
```

Comment:
s/so that it/so it/g

'so that it' is not wrong but removing the 'that' reads better.